docs/reference/settings/notification-settings.asciidoc

```diff
@@ -315,40 +361,48 @@ Specifies account information for sending notifications
 via PagerDuty. You can specify the following PagerDuty account attributes:
 +
 --
-`name`;;
+`name`::
+(<<static-cluster-setting,Static>>)
 A name for the PagerDuty account associated with the API key you
 are using to access PagerDuty. Required.
 
-`secure_service_api_key` (<<secure-settings,Secure>>);;
+`secure_service_api_key`::
+(<<secure-settings,Secure>>)
 The https://developer.pagerduty.com/documentation/rest/authentication[
 PagerDuty API key] to use to access PagerDuty. Required.
 --
 +
-`event_defaults`;;
+`event_defaults`::
```

Comment:
Should this be marked as dynamic?

Reply:
Or rather, this is a dynamic setting in the code, but the below values are marked as static